CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.12.0)
 
 project(ni_grpc_device_server
   LANGUAGES C CXX
-  VERSION 2.9.0)
+  VERSION 2.10.0)
```

Comment:
Resolve comment after https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/859332 goes in